lldb/test/API/python_api/event/TestEvents.py

```diff
@@ -443,11 +474,17 @@ def test_shadow_listener(self):
         # Put in a counter to make sure we don't spin forever if there is some
         # error in the logic.
         counter = 0
+        run_wo_stop = False
```

Comment:
Doesn't look like this is used? Maybe a holdover from a previous implementation?